src/main/java/com/mlaf/hu/proxy/TcpMtpAgent.java

```diff
@@ -44,7 +47,16 @@ public void handleAddedForeignAgent(String address, final String agentName) {
                 amsad.setName(aid);
                 amsad.setState(AMSAgentDescription.ACTIVE);
                 
-                AMSService.register(TcpMtpAgent.this, amsad);          
+                AMSService.register(TcpMtpAgent.this, amsad);   
+                
+                ACLMessage request;
+                try {
+                    request = createInstructionRequest(aid);
+                    send(request);	
+                } catch (ServiceDiscoveryNotFoundException ex) {
+                    Logger.getLogger(TcpMtpAgent.class.getName()).log(Level.WARNING, "DecisionAgent not found, no instructionset requested: {0}", ex);
```

Comment:
Is een "globale" logger geen betere optie? Dan kunnen we ook de loggeragent toevoegen

Reply:
Wat hij zegt.